include/mesh.tcc

```diff
@@ -69,27 +69,26 @@ bool mpm::Mesh<Tdim>::remove_node(
 template <unsigned Tdim>
 template <typename Toper>
 void mpm::Mesh<Tdim>::iterate_over_nodes(Toper oper) {
-  tbb::parallel_for(
-      tbb::blocked_range<int>(size_t(0), size_t(nodes_.size()),
-                              tbb_grain_size_),
-      [&](const tbb::blocked_range<int>& range) {
-        for (int i = range.begin(); i != range.end(); ++i) oper(nodes_[i]);
-      },
-      tbb::simple_partitioner());
+  tbb::parallel_for(tbb::blocked_range<int>(size_t(0), size_t(nodes_.size()),
```

Comment:
Please run through clang-format

Reply:
It should be done already... always do it while compiling. Not sure why it changes...

Reply:
Not sure how you run `clang-format`. One option is to `cd clang-tools` and run `sh ./format.sh`. We use one `.clang-format` file instead of having additional ones in each folder we create.